tracetools/src/tracetools.c

```diff
@@ -489,6 +491,17 @@ DEFINE_TRACEPOINT(
   TRACEPOINT_ARGS(
     buffer))
 
+void __attribute__((constructor)) tracetools_init()
+{
+  if(getenv("TRACETOOLS_DISABLE")) {
```

Comment:
I'd also like to see a simple integration test for the mechanism. Maybe just a simple test in [`test_ros2trace` like this one](https://github.com/ros2/ros2_tracing/blob/d96270a26da9e58f426884753dba0c7a91f84d4b/test_ros2trace/test/test_ros2trace/test_trace.py#L297) that tries to trace some nodes that were launched with the env var and checks that the resulting trace is empty? Unless you have a better idea.

Reply:
That sounds reasonable.

Reply:
Added in 814bdd421e14cb4a6a469719edbe1e39f96fab99

Reply:
Thanks!